pkg/callback/callbacks.go

```diff
@@ -74,52 +81,53 @@ func (c Callback) ID() string {
 	return ""
 }
 
-// CallbackHandler is an interface for an AM callback handler
+// Handler is an interface for an AM callback handler.
 type Handler interface {
-	// Handle the callback by modifying it
-	Handle(cb Callback) error
+	// Handle the callback by modifying it. Return true if the callback was handled.
+	Handle(cb Callback) (bool, error)
```

Comment:
This is the only functional change in the PR. It removed the need for exporting and responding to error variables.